### ISSUE TITLE
Add a way to disable specific rules

### ISSUE THREAD

**Issue by zloirock** [bluwy/publint#249]
With the latest version, I have a false positive
```
$ publint packages/core-js
Running publint v0.3.23 for core-js...
Packing files with `npm pack`...
Linting...
Suggestions:
1. The package publishes internal tests or config files (/modules/es.regexp.test.js). You can use pkg.files to only publish certain files and save user bandwidth.
...
```
I can understand why `publint` suggests that it's a test file - however, it's not a test file, follows `core-js` module naming convention, and renaming this file will break half of the web.

I wanted to use `publint` from CLI with `--strict` - however, the existence of some false positives makes it impossible. Sure, it's possible to write an adapter with the JS API - but it looks like a scratch.

**Comment by bluwy** [comment]
We had request for disabling rules before too, but I'd really like to avoid this as long a possible so that the CLI works ootb with opinionated rules and less configuration.

Would you be able to workaround this by specifying the files field instead? It'd skip the check if that's set. The reason it's based on the field is because npmignore/gitignore is exclude-based and is often the reason unrelated files are accidentally published. Plus, IIRC npm is slowly discouraging using npmignore in favor of the files field.

**Comment by zloirock** [comment]
> Would you be able to workaround this by specifying the files field instead?

It's problematic. There are many packages in the monorepo, some with many thousands of files in different folders with different conventions.

**Comment by bluwy** [comment]
Can you elaborate on that? Do you mean the migration to `"files"` would be tedious?

If you still prefer an exclude based approach, the `"files"` field can work that way like `["*", "!src"]`, I get this is weird because `.npmignore` is basically made for this approach, but just showing some workarounds here.

I'm still hesitant in supporting disabling rules unless there's more rules that may be inaccurate and worth disabling too. Right now, this is the only rule, and in the past this was also the rule that triggered the same request: https://github.com/publint/publint/issues/81.

Alternatively we could add some hardcoded skips for `.test.` files, like `regexp.test` you mentioned that looks like the native API. But I'd consider this the last resort.
